tedana/workflows/tedana.py

```diff
@@ -332,7 +332,7 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
     elif user_tr != 0:
         # Coerce TR to be user-supplied value
         zooms = ref_img.header.get_zooms()
-        new_zooms = (zooms[0],zooms[1],zooms[2],user_tr)
+        new_zooms = (zooms[0], zooms[1], zooms[2], user_tr)
         ref_img.header.set_zooms(new_zooms)
 
         if tr != 0:
```

Comment:
Shouldn't this be 
`if tr != user_tr:`

I can imagine a case where the user is providing a TR, copies that code to another run, and forgets to edit. Thus, the dataset TR is correct (non-zero), but the user is overriding it with an incorrect value.

Reply:
Hm. My logic was that you should throw a warning anytime the TR is nonzero and the user is supplying one, but I guess that your change makes sense because what you *really* care about is a TR modification. I'll modify that. 

This is all predicated though on @tsalo and @emdupre agreeing with adding that option.

Reply:
60f9858 should close this

Reply:
Right, right. I'm good with the warning at 328 - if the header is zero, the user should know - so that takes care of that. But if the header is some_value, and the user provides a different value, then the second warning (line 339) will show up, letting them know that they are forcing the TR to be different - which they may or may not want. 

If the user provides a tr that matches the header, then there is no warning - that seems right to me.